packages/app/src/sections/dashboard/SpotHistoryTable.tsx

```diff
@@ -27,34 +26,39 @@ const conditionalRender = <T,>(prop: T, children: ReactElement) =>
 
 const SpotHistoryTable: FC = () => {
 	const { t } = useTranslation()
-	const { network, walletAddress } = Connector.useContainer()
+	const { network } = Connector.useContainer()
 	const synthsMap = useAppSelector(selectSynthsMap)
 	const { selectPriceCurrencyRate, selectedPriceCurrency } = useSelectedPriceCurrency()
-	const walletTradesQuery = useGetWalletTrades(walletAddress!)
 
 	const synths = useMemo(() => Object.values(synthsMap) || [], [synthsMap])
-	const trades = useMemo(() => {
-		const t = walletTradesQuery.data?.synthExchanges ?? []
 
-		return t.map((trade: any) => ({ ...trade, hash: trade.id.split('-')[0] }))
-	}, [walletTradesQuery.data])
+	const dispatch = useAppDispatch()
+
+	useEffect(() => {
+		dispatch(fetchWalletTrades())
```

Comment:
Be good to use useFetchAction as it involves some extra logic like making sure provider is ready etc before fetching

Reply:
But on second thought I think we want to delete this table, would be good to just clear it out in this PR.

Reply:
Deleted the table.